Source/Core/Core/HW/Memmap.cpp

```diff
@@ -302,12 +302,12 @@ u8* GetPointer(u32 address)
 	// TODO: Should we be masking off more bits here?  Can all devices access
 	// EXRAM?
 	address &= 0x3FFFFFFF;
-	if (address < REALRAM_SIZE)
+	if (address <= REALRAM_SIZE)
```

Comment:
This is a big hack, and in my opinion, broken. if address is equal to REALRAM_SIZE, the pointer will be invalid.